taxi/models.py

```diff
@@ -0,0 +1,31 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+from django.db.models.fields.related import ManyToManyField
+
+from taxi_service import settings
```

Comment:
it's better to use
`from django.conf import settings`
because you may have multiple settings files in the project or folders